lib/puppet/pops/lookup/lookup_adapter.rb

```diff
@@ -85,6 +85,7 @@ def lookup_global(key, lookup_invocation, merge_strategy)
       end
     end
   rescue Puppet::DataBinding::LookupError => detail
+    raise detail unless detail.issue_code.nil?
```

Comment:
I'm curious about this - is the test for `detail.issue_code.nil?` a way we're checking if this is one of our issues described above in `Issues` ?